### PR TITLE
Modernize Bitbucket Pipelines setup

### PR DESCRIPTION
**Pull request by bittner**
Updates the CI/CD setup with Bitbucket Pipelines to

* use Kustomize
* deploy review apps in the Development environment

### Related resources (for reviewer)

* [Same PR on Bitbucket](https://bitbucket.org/painless-software/painless-continuous-delivery/pull-requests/2/modernize-bitbucket-pipelines-setup/diff)
* [Pipelines that ran for the implementation](https://bitbucket.org/vshn/example-bitbucket/addon/pipelines/home)
* APPUiO namespaces for this review: (linked to from Pipeline deployment step)
  - `example-bitbucket-development`
  - `example-bitbucket-integration`
  - `example-bitbucket-production`

### CODE REVIEW

File: {{cookiecutter.project_slug}}/_/ci-services/deploy-stage/bitbucket-pipelines.yml [hunk 1]
```diff
@@ -1,23 +1,105 @@
...
+        --from-literal=POSTGRESQL_DATABASE=${DATABASE_NAME}
+        --from-literal=POSTGRESQL_USERNAME=${DATABASE_USER}
+        --from-literal=POSTGRESQL_PASSWORD=${DATABASE_PASSWORD}
 
-  branches:
-    master:
-    - parallel: *checks
-    - parallel: *tests
+  - step: &deploy-review-app
+      name: Deploy Review App
```
Comment (srueg) [inline {{cookiecutter.project_slug}}/_/ci-services/deploy-stage/bitbucket-pipelines.yml]: Where/how is the review app deleted again?

Comment (bittner) [reply {{cookiecutter.project_slug}}/_/ci-services/deploy-stage/bitbucket-pipelines.yml]: You can see it for the output of the deployment step. There are links to click on.

Care to find out yourself? :smiley_cat: :crossed_fingers:

Comment (bittner) [reply {{cookiecutter.project_slug}}/_/ci-services/deploy-stage/bitbucket-pipelines.yml]: Sorry, misread your question. There are instructions in the deployment steps. No automation possible. We could implement a "custom" action.

Comment (srueg) [reply {{cookiecutter.project_slug}}/_/ci-services/deploy-stage/bitbucket-pipelines.yml]: Ah, I see. Well too bad.

**Pull request merged**